app/cdash/tests/test_donehandler.php

```diff
@@ -31,6 +31,11 @@ public function testDoneHandlerLocal()
 
     public function testDoneHandlerRemote()
     {
+        if (config('filesystem.default') !== 'local') {
+            // Skip this test case if we're already testing remote storage.
+            return;
```

Comment:
I believe the preferred way to do this in PHPUnit is to use `$this->markTestSkipped(...)`

Reply:
Unfortunately this is an old style "simpletest", not a PHPUnit test. It does look like simpletest has a `skipIf` method, but playing around with it locally, it doesn't actually stop the rest of the test from executing??